### ISSUE TITLE
Starmade occaissonally crashes to desktop upon entering or exiting WarpSpace

### ISSUE THREAD

**Issue by NotAVerb** [IR0NSIGHT/Warpspace#15]
I think I've found the right log. It's a stack overflow.
[logstarmade.4.log](https://github.com/IR0NSIGHT/Warpspace/files/6598137/logstarmade.4.log)

**Comment by IR0NSIGHT** [comment]
Does this happen every time you reach your dropout point while having a waypoint assigned? Or is it completely random?

**Comment by NotAVerb** [comment]
Random sadly. I was able to reach my destination several times mere moments ago.

**Comment by NotAVerb** [comment]
Another log where it crashed. Another ugly wall of null waypoints followed by a stack overflow.
[logstarmade.9.log](https://github.com/IR0NSIGHT/Warpspace/files/6598263/logstarmade.9.log)

For completion's sake, here's my unsuccessful attempt at crashing, where everything worked as expected albeit the background was still borked.
[logstarmade.0.log](https://github.com/IR0NSIGHT/Warpspace/files/6598265/logstarmade.0.log)

**Comment by IR0NSIGHT** [comment]
found a way to trigger it. sitting in your warp dropout point and setting your own waypoint to nothing in nav console triggers an infitinte loop which crashes the game

**Comment by NotAVerb** [comment]
I'm racking my brain trying to figure out when I did something like that. I'm wondering if I tried setting a new destination while dropping out. I'm the sort to engage warp, then use a saved nav point after the fact. In fact I'm going to test if that crashes right now.

Just got another crash after jumping in without a set waypoint. I was able to do it once before without issue, and then set 2 2 2 as a destination. Second time crashed, however I opened the chat window after engaging warp so maybe that was it?
[logstarmade.0.log](https://github.com/IR0NSIGHT/Warpspace/files/6598336/logstarmade.0.log)
